.github/ISSUE_TEMPLATE/roadmap.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Roadmap issue
+about: Use this template for tracking release roadmaps.
+title: "Roadmap for vXXXXX"
+labels: roadmap
+assignees: zimmski
+---
+
```

Comment:
Do a goal-summary on what we want to do for the release: reasons for more available scores.

Reply:
- Goals are based on "main topics" (at least one) towards extending the benchmark on a score-base
- A new tool, a new model, ... is not release-worthy, but it is publish-worthy i.e. how is that new model doing with the benchmark?

Reply:
Content for a deep-dive blog post:
- Content is about giving others an insight of our experiences: everyone who reads, should learn something
- Findings / Problems. if you are surprised, it is the best content!
  - Journal about what went well
  - Journal about what went not well
- For every finding immediately write down a few bullet points why it is interesting, give examples(!)